### ISSUE TITLE
Plugin Benennung: "Commons Booking" oder "CommonsBooking"?

### ISSUE THREAD

**Issue by flegfleg** [wielebenwir/commonsbooking#881]
Plugin-Name: "Commons Booking"
Settings-Eintrag: "CommonsBooking"

-> an sich heisst das Ding "CommonsBooking", oder nicht?

**Comment by chriwen** [comment]
@flegfleg Ja, das hatten wir mal auf deinen Vorschlag hin auseinander geschrieben, da sonst der Plugin-Name im Menü seltsam abgeschnitten wird...

**Comment by flegfleg** [comment]
Ja, jetzt erinnere ich  mich, hatte damals schon kommentiert: aber nicht im Plugin-Namen, nur das Menü.
Wirkt bei einer Migration irritierend, diese CommonsBookings :)
